test/Source/Fixtures/ObjectWithToString.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Exporter\Test\Source\Fixtures;
+
+/**
+ * @author Joseph Maarek <josephmaarek@gmail.com>
+ */
+final class ObjectWithToString
+{
+    private $name;
+
+    public function __construct($name)
+    {
+        $this->name = $name;
+    }
+
+    public function __toString()
```

Comment:
This is a question also: does __toString() accept return type string?

IMO it should have it, but not sure if it works as expected

Reply:
Yes: https://3v4l.org/PppGl

Reply:
Not relevant anymore, 1.x will still support PHP 5.6.